source/_integrations/zha.markdown

```diff
@@ -255,23 +255,9 @@ custom_quirks_path:
 
 ### Advanced OTA configuration
 
-The default configuration for OTA firmware updates is chosen by ZHA developers, so normal users should not need to change any configuration. Most of the config options listed in the zigpy section are just meant for development or advanced users.
+OTA for a few manufacturers is enabled by default, currently Ledvance, Sonoff, Inovelli, and ThirdReality. Other manufacturers are supported but disabled by default because their updates may change or remove device functionality, may require you to reconfigure devices, or are contributed by the community and may be minimally tested.
 
-Further advanced configuration options are only provided in the [zigpy project's developers documentation](https://github.com/zigpy/zigpy).
-
-However, if you want to disable OTA updates for a specific manufacturer, you can add the following lines to your `configuration.yaml` and restart Home Assistant.
-
-```yaml
-zha:
-  zigpy_config:
-    ota:
-      ikea_provider: false                       # Disable OTA update downloads for Trådfri devices
-      inovelli_provider: false                   # Disable OTA update downloads for INOVELLI devices
-      ledvance_provider: false                   # Disable OTA update downloads for LEDVANCE/OSRAM devices
-      salus_provider: false                      # Disable OTA update downloads for SALUS/Computime devices
-      sonoff_provider: false                     # Disable OTA update downloads for Sonoff (ITead) devices
-      thirdreality_provider: false               # Disable OTA update downloads for 3REALITY devices
-```
+Refer to the [zigpy documentation for OTA configuration](https://github.com/zigpy/zigpy/wiki/OTA-Configuration) for more information on additional OTA providers.
```

Comment:
I'm not a fan of sending end-user into GitHub pages for documentation. If the information on that page is relevant, why not include it in the main documentation?

../Frenck

Reply:
It's "advanced" documentation that should be modified with caution. The issue before was that the example from HA docs was often copied 1:1. This caused additional support requests and bricked devices in some cases.

There are other/better enabled-by-default OTA options planned in the future, but these options shouldn't need to be touched in most cases. The compromise is linking to the zigpy wiki. It contains more information for advanced users, but it's (hopefully) less likely just to get its examples pasted into the HA config.

Alternatively, we could also remove the link to the zigpy wiki completely and not include the OTA config example in the docs by default. Users that really want/need it would search outside HA docs.

I think we prefer if there's no OTA config example (that can cause issues for inexperienced users) in the docs. Most OTA providers are enabled by default and custom "verified" ones, like Z2M somewhat has, can be added in the future and require no user configuration. (Just an update of this docs text mentioning which OTA providers are active.)

Of course, if you think it’s best to re-add the (updated) example, the ones from the wiki can be slimmed down (with the dangerous options documented even more), then added to these docs.

(Also, throughout the years, the ZHA docs page grew rapidly and is IMO too long at this point. It’s why we try to keep only crucial information. It’s an ongoing effort to update the docs page and make it more clear.)

This comment turned out a bit longer than expected 😅, but let me know what you think what you think is best.

Reply:
I agree with @TheJulianJES. I'd ideally want to remove this documentation section from ZHA entirely because it's a remnant of ZHA prior to us implementing the `update` platform and has the potential for dangerous configurations (e.g. IKEA pushing bricking updates, image in the Z2M repo not having adequate metadata to restrict it to the correct device).

Many people have copy/pasted the entire config and changed `false` to `true`. We've moved our OTA configuration to a different format and are actively logging deprecation warnings (including an explanation of how to fix each one). Linking to the zigpy documentation seems like the best option, in my opinion, as I do not want to reproduce it in the ZHA OTA section.